Makefile

```diff
@@ -48,5 +48,7 @@ clean:
 update-project-template:
 	npx update-template https://github.com/googlefonts/googlefonts-project-template/
 
-update:
-	pip install --upgrade $(dependency); pip freeze > requirements.txt
+update: venv
+	venv/bin/pip install --upgrade pip-tools
+	venv/bin/pip-compile --upgrade --resolver=backtracking requirements.in
```

Comment:
what does --resolver=backtracking do?

Reply:
Not sure, but pip-compile prints a warning that it's becoming the default in the next version and that you should pass the flag.